build.gradle

```diff
@@ -77,6 +77,10 @@ allprojects { subproj ->
 	apply plugin: 'com.github.ben-manes.versions' // gradle dependencyUpdates
 	apply from: "$rootDir/gradle/degraph.gradle"
 
+	tasks.withType(Test) { task ->
+		task.testLogging.exceptionFormat = 'full'
+	}
+
```

Comment:
That's to debug the error. The CI job didn't have a whole lot of info about what happened so I was hoping to get more logging/stacktrace. I'll remove it before merging.